### PR TITLE
obsidian plugin update

### PR DESCRIPTION
**Pull request by sywhb**
- fix: use multiline string for title and author in default template #49 
- fix: add wordsCount and readLength to the exposed variables in the article template #56

### CODE REVIEW

**Comment by sywhb** [comment]
fix #47

File: src/main.ts [hunk 1]
```diff
@@ -39,6 +40,21 @@ export default class OmnivorePlugin extends Plugin {
...
+    // update version if needed
+    const latestVersion = this.manifest.version;
+    const currentVersion = this.settings.version;
+    if (latestVersion !== currentVersion) {
+      this.settings.version = latestVersion;
+      this.saveSettings();
+      // show release notes
+      const releaseNotes = `Omnivore plugin is upgraded to ${latestVersion}.
+    
+    What's new: https://github.com/omnivore-app/obsidian-omnivore/blob/main/CHANGELOG.md
```
Comment (jacksonh) [inline src/main.ts:53]: nice

**Comment by sywhb** [comment]
fix: #55

**Comment by sywhb** [comment]
fix: #31 and #33

**Comment by sywhb** [comment]
fix #63

**Pull request merged**